cmd/user.go

```diff
@@ -49,10 +49,11 @@ users:
 
 	// userCmd creates new certs and kubeConfig for a user
 	userCmd = &cobra.Command{
-		Use:   "user",
+		Use:   "user [command]",
```

Comment:
I know it isn't related to this PR but the command name is a little misleading here. As the command is described as "Manage user access" but in fact it just creates a kubeconfig.

Reply:
I believe this stems from the original issue: #389 and perhaps should be better clarified in a future PR.